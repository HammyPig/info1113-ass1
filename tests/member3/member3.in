member 100000
ADD MEMBER gibbeh
MEMBER 100000