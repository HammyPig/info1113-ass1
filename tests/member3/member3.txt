recognises if no members are present