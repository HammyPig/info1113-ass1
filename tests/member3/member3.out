user: No members in system.

user: Success.

user: 100000: gibbeh

user: Ending Library process.
